src/coreclr/gc/gc.cpp

```diff
@@ -11541,6 +11532,477 @@ void gc_heap::rearrange_heap_segments(BOOL compacting)
 }
 #endif //!USE_REGIONS
 
+#if defined(USE_REGIONS)
+// trim down the list of free regions pointed at by free_link down to target_count, moving the extra ones to surplus_link
```

Comment:
comment still calls it link instead of list

Reply:
Yup, thanks!